### PR TITLE
Hamburger menu Interactions

### PR DESCRIPTION
**Pull request by sharon-odhiambo**
In this project we did the following:

Style the hamburger menu and add exit button.
Use Javascript events to add Interactions to the hamburger menu
Use JS events to add interactions to mobile menu items.

### CODE REVIEW

**Comment by thecodechaser** [comment]
@sharon-odhiambo Please attach a screenshot of linter check in your local machine, As you said there's no error locally

File: index.html [hunk 1]
```diff
@@ -20,19 +20,19 @@
...
       <div class="mobile-navbar" id="mobile-navbar">
+        <img class="exit-btn" width="18" height="16" src="./images/cancel-icon.png" alt="hamburger cancel button">
         <ul class="mobile-nav-menu" id="mobile-nav-menu">
-          <li class="mobile-nav-link"><a href="#works">Portfolio</a></li>
-          <li class="mobile-nav-link"><a href="#about-myself">About</a></li>
-          <li class="mobile-nav-link"><a href="#form-section">Contact</a></li>
+          <li class="mobile-nav-link"><a href="#works-sect">Portfolio</a></li>
+          <li class="mobile-nav-link"><a href="#about-me">About</a></li>
+          <li class="mobile-nav-link"><a href="#form">Contact</a></li>
         </ul>
```
Comment (ShahierNashaat) [inline index.html:28]: - [ ] It is better to wrap the x button and the menu items in a div and give it background: rgb(0, 0, 255) and opacity: 0.6 to make the mobile menu looks like the Figma design.

Your Design | Figma Design
---------------- | --------------------
Screenshot from 2022-06-21 16-15-41 | Screenshot from 2022-06-21 16-09-48

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly be noticed that there is a horizontal scroll on your website please fix that to make your website responsive.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -20,19 +20,21 @@
...
+        </div>
         <div class="menu-button">
           <img class="menu-button" width="18" height="12" src="images/Union.svg" alt="menu-icon">
         </div>
       </div>
     </nav>
   </header>
-
-  <section class="headline" id="headline"> 
+  <div class="headline" id="headline">
```
Comment (danmainah) [inline index.html:37]: [ ] [optional] @sharon-odhiambo great work implementing the changes, kindly check on your design the background images are not in the desired position like in the Figma design:
image

**Review by danmainah — APPROVED** [review]
Hi @sharon-odhiambo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**